caas/kubernetes/provider/admissionregistration.go

```diff
@@ -23,18 +24,31 @@ func (k *kubernetesClient) getAdmissionControllerLabels(appName string) map[stri
 	}
 }
 
+const (
+	annotationNoPrefixingKey   = "no-prefixing"
```

Comment:
Maybe "disable-name-prefix"
Since "no-prefixing" doesn't really say what it is

Reply:
Just changed it to `juju.io/disable-name-prefix`;